learning-outcomes.md

```diff
@@ -0,0 +1,50 @@
+# Week 4 Learning Outcomes
+
+## Node
+
+### Servers
+
+- [ ] **We can create a web server that responds to http requests**
+- [ ] We can conditionally set ports for our server based on the runtime environment
+- [ ] We can read secrets from a .env file
+- [ ] We can prevent sensitive data from being pushed to GitHub
+
+### GET requests
+
+- [ ] We can route requests to the correct handler function(s)
+- [ ] We can serve different types of files to the client
+- [ ] We can read information sent in a URL query string
+
+### POST requests
```

Comment:
The `GET` section is stuff that applies to any HTTP method really. Most of the `POST` stuff is about handling HTTP bodies, which lots of methods can have.

Could we instead put the `GET` stuff under "Servers" and have the specific "submitting data" LOs under "Request bodies" or something?